.github/templates/main.yml

```diff
@@ -79,6 +79,20 @@ steps:
 #@ end
 #@ end
 
+#@ def buildUnityPackage(includeDependencies):
+#@ finalPkgName = "io.realm.unity-" + ("bundled-" if includeDependencies else "") + "${{ steps.find-nupkg-version.outputs.package_version }}.tgz"
+  - name: #@ "Build Unity" + (" Bundled Dependencies" if includeDependencies else "")
+    run: #@ "dotnet run --project Tools/SetupUnityPackage/ -- realm --packages-path Realm/packages --pack" + (" --include-dependencies" if includeDependencies else "")
+  - name: Store Unity artifacts
+    uses: actions/upload-artifact@v2
+    with:
+      name: #@ finalPkgName
+      path: #@ "${{ github.workspace }}/Realm/Realm.Unity/" + finalPkgName
+      retention-days: ${{ github.event_name != 'pull_request' && 30 || 1 }}
+  - name: Cleanup build folder
+    run: #@ "rm ${{ github.workspace }}/Realm/Realm.Unity/" + finalPkgName
```

Comment:
Why do you need to clean the build folder?
I understand that you call this function twice (line 267 and 268) but the final package of each call has a different name. What am I missing?